TESTING-INSTRUCTIONS.md

```diff
@@ -409,6 +409,23 @@ Scenario #2
 8. Click on the **Choose payment methods** task, it should not be displaying the **Woocommerce Payments** option.
 9. Go to **Plugins > installed Plugins**, check if the selected plugin features selected in step 4 are installed and activated.
 
+### Improve AddFirstProduct email note contents #6617
+
+- Install the plugin in a fresh site site.
```

Comment:
Just to be extra safe, might be worth mentioning that the store should have 0 orders and 0 products.

Also `site site` 😄